plugins/module_utils/opennebula.py

```diff
@@ -26,6 +26,52 @@
     HAS_PYONE = False
 
 
+# A helper function to mitigate https://github.com/OpenNebula/one/issues/6064.
+# Used for template merging in the JSON-like template representation.
+def extend(to_extend, extend_by):
```

Comment:
You might want to take a looks at `ansible.module_utils.common.dict_transformations.dict_merge()`, I think it does the same thing.

Reply:
Well, almost... :sweat_smile: , but the merge itself is the same. I adapted the `dict_merge` function, thank you. :hugs: